core/src/main/java/org/elasticsearch/index/query/QueryBuilders.java

```diff
@@ -35,9 +35,12 @@
 import java.util.List;
 
 /**
- * A static factory for simple "import static" usage.
+ * Utility class to create search queries.
  */
-public abstract class QueryBuilders {
+public class QueryBuilders {
```

Comment:
make it also final just to make it clearer? Although the private constructor says it all.